kyo-core/jvm/src/test/scala/kyoTest/completionStageTest.scala

```diff
@@ -0,0 +1,14 @@
+package kyoTest
+
+import java.util.concurrent.*
+import kyo.*
+import kyo.Fibers
+
+class completionStageTest extends KyoTest:
+    "completionStage to Fiber" in runJVM {
```

Comment:
Could you add a test with a failure and another with locals? For locals, maybe something along these lines:

```scala
val local = Locals.init[Int](0)
Locals.let(42) {
   Fibers.fromCompletionStage(...).map { _ =>
       assert(local.get == 42)
   }
}
```

Reply:
yessir